api/client/samples/anomaly_detection/sample_anomaly_detection.ipynb

```diff
@@ -0,0 +1,793 @@
+{
```

Comment:
<p><code>is_max</code> variable not used. Also, the min condition is also using the <code>is_max</code> variable. Should max and min both set <code>is_significant = True</code> too?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gro-intelligence/api-client/pull/146/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>ah yes this was me combining multiple functions - good catch</p>